src/styles/fields.styl

```diff
@@ -18,6 +18,15 @@
     .password-reset-link
         margin-top      .5em
 
+    .coz-form-controls
+        padding 0
+        margin  1em 0
```

Comment:
Just review the order of the properties, from the outside to the inside, in this case : 
margin -> width -> padding
and then , just align the values and we're good ^_^